src/core/css/SearchResult.scss

```diff
@@ -59,6 +60,8 @@
 
 .SearchResult-author {
   margin-right: 0.5em;
+  word-wrap: break-word;
+  display: block;
```

Comment:
Generally we try to alpha-sort descriptors e.g. `display` before `margin...`. It's not always appropriate - which makes it difficult to use a style-lint rule to automatically flag it - but generally it does aid readability.